privacyidea/lib/tokens/hotptoken.py

```diff
@@ -351,14 +353,16 @@ def update(self, param, reset_failcount=True):
 
         # check if the key_size is provided
         # if not, we could derive it from the hashlib
-        key_size = getParam(upd_param, 'key_size', optional) \
-                   or getParam(upd_param, 'keysize', optional)
+        key_size = upd_param.get('key_size') or upd_param.get('keysize')
```

Comment:
I think we can get rid of `key`size` here, the parameter name is `keysize` as described in the token API docs